src/docs/primitives/a-model.md

```diff
@@ -29,3 +29,9 @@ Using scale to increase to size of the loaded tree model:
 ```html
 <a-model scale="10 10 10" src="https://aframe.io/aframe/examples/_models/tree1/tree1.dae"></a-model>
 ```
```

Comment:
I think adding more examples is fine. Otherwise, we'd be duplicating a lot of tables. Linking from each component mapping in the primitive table to the component definition table should help as well.